app/models/mixins/cinder_manager_mixin.rb

```diff
@@ -18,24 +18,10 @@ module CinderManagerMixin
     private
 
     def ensure_cinder_managers
```

Comment:
This is a bit of a weird name since it sounds like it is working on multiple cinder managers but only ever works on one.

Reply:
this is following some kind of naming convention

Reply:
Well the convention usually is a top-level `ensure_managers` calls e.g. `ensure_network_manager` and `ensure_storage_manager`